backend/src/plugins/ModActions/functions/parseReason.ts

```diff
@@ -0,0 +1,12 @@
+const MAX_REASON_LENGTH = 512;
+
+export function parseReason(config: any, reason: string): string {
+  if (!reason) return reason;
+  if (config?.reason_aliases) {
+    reason = config.reason_aliases![reason.toLowerCase()] ?? reason;
```

Comment:
If you're making the `reason` lowercase here, you should make the `reason_aliases` keys lowercase too